app/Http/Controllers/HomeController.php

```diff
@@ -2,11 +2,37 @@
 
 namespace App\Http\Controllers;
 
+use Cache;
+use App\User;
+use App\Models\Thread;
+
 class HomeController extends Controller
 {
     public function show()
     {
-        return view('home');
+        $totalUsers = Cache::remember('totalUsers', now()->addDay(), function () {
+            return number_format(User::count());
+        });
+
+        $totalThreads = Cache::remember('totalThreads', now()->addDay(), function () {
+            return number_format(Thread::count());
+        });
+
+        $resolutionTime = Cache::remember('resolutionTime', now()->addDay(), function () {
+            return number_format(Thread::resolutionTime());
+        });
+
+        $latestThreads = Thread::feedQuery()
+            ->whereNull('solution_reply_id')
+            ->limit(3)
+            ->get();
```

Comment:
Maybe it's worth to rethink this in the future so it takes some random unresolved questions from the last week. And maybe also cache the result for like an hour or so. Before this, the homepage didn't use any query at all :)